AnnoDesigner/Models/LayoutObject.cs

```diff
@@ -374,8 +374,22 @@ public double GetScreenRadius(int gridSize)
         {
             if (_screenRadius == default || _lastGridSizeForScreenRadius != gridSize)
             {
-                _screenRadius = _coordinateHelper.GridToScreen(WrappedAnnoObject.Radius, gridSize);
-
+                // Buildings of an odd-numbered size (3x3/3x5 etc) draw their circular influence range with an additional +0.5, this is not correct and needs
+                // to be adjusted to produce the correct radius for those buildings.
+                
+                // To get Round up numbers, like 1.5 becomes 1.0 , 2.25 becomes 2.00
```

Comment:
This isn't needed - instead of dividing by 2 and then rounding the values down then comparing to the original values divided by 2, we can just use the modulo  (`%`) operator, e.g:

`true` = even, `false` = odd
```csharp
WrappedAnnoObject.Size.Width % 2 == 0 
```

Reply:
i did not know about operators like that, is there a site where i can look up that kind of stuff ?

Reply:
There's always the c# reference, which isn't exactly a great tutorial, but will tell you about absolutely everything: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/arithmetic-operators#remainder-operator-

And now I learn that `%` is actually the "remainder" operator in C#. You'll hear it be called "modulo", "modulus", "remainder", and usually just the "mod" operator. Above, we're effectively just finding the remainder when the value is divided by 2. 

It occurs to me that I'm assuming that width and height are always integer values - if they had decimal parts (e.g 1.3, 4.87), you'd end up with a result less than 1 for an "even" number (or at least where the integer part of the number is even) and more than or equal to 1 for an odd number. For our use case though, width and height should always be integers.

Reply:
Oke i will read me in later today, and see of i'm able to change the stuff in this case